com.zsmartsystems.zigbee/src/main/java/com/zsmartsystems/zigbee/ZigBeeEndpoint.java

```diff
@@ -501,15 +501,25 @@ public void setDao(ZigBeeEndpointDao dao) {
         if (dao.getInputClusterIds() != null) {
             for (ZclClusterDao clusterDao : dao.getInputClusters()) {
                 ZclCluster cluster = getClusterClass(clusterDao.getClusterId());
-                cluster.setDao(clusterDao);
-                inputClusters.put(clusterDao.getClusterId(), cluster);
+                if (cluster != null) {
+                    cluster.setDao(clusterDao);
+                    inputClusters.put(clusterDao.getClusterId(), cluster);
+                } else {
+                    logger.debug("Manufacturer specific input cluster found with id={}, will skip it",
```

Comment:
Maybe we should just log "Unknown input cluster  found with id={}....." as we don't necessarily know it's a manufacturer specific cluster - just that it's a cluster that can't be created.